src/lib/components/FileUpload/FileUpload.scss

```diff
@@ -2,13 +2,16 @@
 @include vf-base;
 @include vf-p-buttons;
 
+.file-upload__wrapper {
+  padding-bottom: $spv--medium;
+}
+
 .file-upload {
-  background: linear-gradient(to right, $color-mid-light 50%, transparent 50%) 0 0,
-    linear-gradient(to right, $color-mid-light 50%, transparent 50%) 0 100%,
-    linear-gradient(to bottom, $color-mid-light 50%, transparent 50%) 0 0,
-    linear-gradient(to bottom, $color-mid-light 50%, transparent 50%) 100% 0;
-  background-repeat: repeat-x, repeat-x, repeat-y, repeat-y;
-  background-size: 10px $input-border-thickness, 10px $input-border-thickness, $input-border-thickness 10px, $input-border-thickness 10px;
+  border: 0.125rem solid $color-transparent;
```

Comment:
Could you please add a basic border CSS transition to match vanilla framework design (e.g. notice regular button hover transitions).